highfive/configs/rust-lang/rust.json

```diff
@@ -133,6 +133,10 @@
         "src/tools/tidy":                        ["@Mark-Simulacrum"]
     },
     "mentions": {
+        "library/portable-simd": {
+            "message": "library/portable-simd is a subtree of https://github.com/rust-lang/portable-simd and changes here will have to be cherry-picked upstream.",
```

Comment:
This message reads a little to me like it's suggesting that either (a) the change should not be made in rust-lang/rust or (b) that the author is responsible for cherry-picking upstream. I think neither of those are generally true, so I wonder if we can try to figure out a slightly clearer message, maybe just by tagging on a "This is something the project group will take care of, so you don't need to worry about it." or similar.

Reply:
To be clear, from this perspective, many of our messages here are not great, it's just that this is being added and so is coming under slightly more scrutiny by me :)

Reply:
tbh, it is *somewhat* preferred if it can be avoided! But I agree that it is Not Necessarily True, just a weak preference in the case where there's no other reason not to, so I am fine with changing the message. The real thing that would be consistently desired, I think, would be that the commits should be structured so that cherry-picking the diff out is easy (it is much more annoying if the same commit diffs both library/portable-simd and the rest of the tree, and structuring a PR in certain ways, like preferring rebasing, is already imposed on contributors) and yes, that it needs to alert someone to do that.

I would rather Insinuate Something, even if that could be misinterpreted, than the more... nondescript quality of most of these messages. Presenting a total void of context is more alarming, in my experience as a contributor, than implying something about preferred processes. A suggestive direction points towards what might be more ideal, and even if misinterpretable, increases confidence about directions. The more ambiguous messages accept any interpretation, decreasing confidence about any direction, almost to the point of inducing anxiety.